docs/1manifoldLearning.md

```diff
@@ -0,0 +1,53 @@
+# 1-Manifold Learning example
+
+![1-Manifold Learning example Image](https://topology-tool-kit.github.io/img/gallery/1manifoldLearning.jpeg)
+
+
+## Pipeline description
+
+This example first loads cloud of points with elevation values from disk. 
```

Comment:
cloud of points -› a point cloud.
don't mention the elevation (I don't know why it's there :)  and this is not even used in the pipeline)